src/components/Navbar.jsx

```diff
@@ -16,7 +16,7 @@ const Navbar = () => {
 
   let renderCategories = categories.map((cat) => {
     return (
-      <Menu.Item>
+      <Menu.Item boderless>
```

Comment:
borderless typo?

Reply:
styling, chore

Reply:
But is it a typo and should say "borderless", because it doesn't

Reply:
yep, whoops, fixed!